### DISCUSSION TITLE
404 not found

### DISCUSSION THREAD

**Discussion by enzofoucaud** [disgoorg/disgo#313] [Help] [answered]
**Describe the bug**
404 not found error

**Error**
The error which gets logged if there is any:
```
ERROR error sending message: Status: 404 Not Found, Body: {"message": "404: Not Found", "code": 0}
```

**To Reproduce**
```go
	if _, err := clientDiscord.CreateMessage(discord.NewWebhookMessageCreateBuilder().
		SetContentf("test 1").
		Build(),
		// delay each request by 2 seconds
		rest.WithDelay(2*time.Second),
	); err != nil {
		log.Errorf("error sending message: %s", err)
	}
```

**Disgo Version(please complete the following information):**
 - Version 0.16.9

**Comment by topi314** [comment, marked as answer]
this suggests the webhook you're trying to use does not exist, make sure you put the right id & token in
